models/blog/schema.gadget.ts

```diff
@@ -9,6 +9,7 @@ export const schema: GadgetModel = {
     createdAt: { type: "DateTime", validations: [{ type: "required" }], dataKey: "mf-sys-ofO4RD-createdAt" },
     description: { type: "String", validations: [{ type: "required" }], dataKey: "mf-Ms3dOA" },
     id: { type: "ID", validations: [{ type: "required" }, { type: "unique" }], dataKey: "mf-sys-bPCqjx-id" },
+    state: { type: "Enum", default: "published", options: ["draft", "published"], dataKey: "mf-nMx5pI" },
```

Comment:
default should be `draft` if the intent is for creating a new blog (`saveAsDraft`) to be created in draft mode

Reply:
nit: use all caps for enum values

Reply:
should also be required